### PR TITLE
Fix IT tests

### PR DESCRIPTION
**Pull request by cristianonicolai**

### CODE REVIEW

**Review by nmirasch — APPROVED** [review]

**Review by radtriste — APPROVED** [review]

**Comment by radtriste** [comment]
Jenkins retest this

File: data-index/data-index-service/data-index-service-infinispan/pom.xml [hunk 1]
```diff
@@ -97,13 +97,13 @@
...
           <execution>
-            <id>default</id>
+            <id>integration-test</id>
             <goals>
               <goal>integration-test</goal>
             </goals>
             <configuration>
               <excludes combine.children="append"> <!-- Add the exclude to existing ones -->
-                <exclude>**/*Keycloak*IT.java</exclude> <!-- Separate execution for this one -->
+                <exclude>**/Keycloak*IT.java</exclude> <!-- Separate execution for this one -->
```
Comment (danielezonca) [inline data-index/data-index-service/data-index-service-infinispan/pom.xml]: Is this change needed? I makes the configuration very specific 🤔 
As far as I remember we usually exclude everything like `Native*IT` or similar

File: data-index/data-index-service/data-index-service-mongodb/pom.xml [hunk 2]
```diff
@@ -91,13 +91,13 @@
...
           <execution>
-            <id>default</id>
+            <id>integration-test</id>
             <goals>
               <goal>integration-test</goal>
             </goals>
             <configuration>
               <excludes combine.children="append"> <!-- Add the exclude to existing ones -->
-                <exclude>**/*Keycloak*IT.java</exclude> <!-- Separate execution for this one -->
+                <exclude>**/Keycloak*IT.java</exclude> <!-- Separate execution for this one -->
```
Comment (danielezonca) [inline data-index/data-index-service/data-index-service-mongodb/pom.xml]: Same as above

File: data-index/data-index-service/data-index-service-infinispan/pom.xml (hunk 1 above)
Comment (cristianonicolai) [reply data-index/data-index-service/data-index-service-infinispan/pom.xml]: not really necessary, just that it became the standard, will revert

**Review by danielezonca — APPROVED** [review]

**Comment by danielezonca** [comment]
jenkins retest this please

**Comment by cristianonicolai** [comment]
jenkins please retest this

**Comment by danielezonca** [comment]
jenkins retest this please

**Comment by danielezonca** [comment]
jenkins retest this please
